pkgs/games/jazz2/content.nix

```diff
@@ -0,0 +1,25 @@
+{ lib
+, jazz2
+, stdenv
+}:
+
+stdenv.mkDerivation rec {
+  pname = "jazz2-content";
+
+  inherit (jazz2) version src;
+
+  dontBuild = true;
+
+  installPhase = ''
+    cp -r Content $out
+  '';
```

Comment:
We could just use runCommand here

Reply:
I'm not really sure how I'm supposed to do that 😕